safetensors/src/tensor.rs

```diff
@@ -833,6 +838,34 @@ mod tests {
         let _parsed = SafeTensors::deserialize(&out).unwrap();
     }
 
+    #[test]
+    fn test_empty() {
+        let tensors: HashMap<String, TensorView> = HashMap::new();
+
+        let out = serialize(&tensors, &None).unwrap();
+        assert_eq!(
+            out,
+            [8, 0, 0, 0, 0, 0, 0, 0, 123, 125, 32, 32, 32, 32, 32, 32]
+        );
```

Comment:
Why does calling `serialize` on an empty HashMap result in a non zero length array?